phpcs.xml.dist

```diff
@@ -34,5 +34,6 @@
   <exclude-pattern>*/node_modules</exclude-pattern>
   <exclude-pattern>*/vendor</exclude-pattern>
   <exclude-pattern>*/default.local.settings.php</exclude-pattern>
+  <exclude-pattern>*md</exclude-pattern>
```

Comment:
There is a section above which may be a better fit...?
```xml
<arg name="ignore" value="*.css,*.md,*.txt"/>
```

Reply:
I wonder if this pattern is too broad. I'd have to verify, but I expect that a directory named `shazamd` would also have its contents ignored with this. As dpagini pointed out, previous sections use a `*.md` pattern.

Reply:
I've confirmed by suspicion. With the `  <exclude-pattern>*md</exclude-pattern>` rule, if I put an improperly formatted css file inside a directory named `shabamd` (or anything else ending in md, phpcs stops seeing the css file. If I then rename the directory to `shabam`, phpcs sees the css file again.